distribution/tools/entitlements-agent/build.gradle

```diff
@@ -0,0 +1,48 @@
+import static java.util.stream.Collectors.joining
+
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+apply plugin: 'elasticsearch.build'
+
+base {
+  archivesName = 'elasticsearch-entitlements-agent'
```

Comment:
Let's not do this unless there's a compelling reason for the `elasticsearch` prefix. It just makes things so confusing since the artifact doesn't match the project name, etc.

Reply:
I can delete this entirely if you want? I only added it in a flurry of troubleshooting.

Reply:
You can remove, and simply refer to this project as `entitlements-agent` instead of `elasticsearch-entitlements-agent`. I don't think it's necessary to prefix _everything_ with "elasticsearch". I don't think that addition context is striclty necessary ;)

Reply:
Ok sounds good. I was just cargo-culting stuff I found elsewhere. Good to have an informed opinion instead. 😅